lib/travis/guest-api/app/endpoints/attachments.rb

```diff
@@ -0,0 +1,19 @@
+require 'travis/guest_api'
+require 'travis/guest-api/app/base'
+
+class Travis::GuestApi::App::Endpoints
+  class Attachments < Travis::GuestApi::App::Base
+
+    before do
+      @job_id = env['job_id'] ? env['job_id'] : params['job_id']
+      halt 422, { error: 'Job ID must be specified.'} unless @job_id
+    end
```

Comment:
now it is ok, but please create a "refactoring" commit for do this `before` block generally for all endpoints.